anesthetic/plot.py

```diff
@@ -14,12 +14,14 @@
 import pandas
 import matplotlib.pyplot as plt
 from matplotlib.gridspec import GridSpec as GS, GridSpecFromSubplotSpec as SGS
+from astropy.visualization import hist
```

Comment:
``astropy`` is an excellent package, although it is a pretty heavyweight one. Given that it's only usage here is for using it's more flexible ``hist`` function, it would be good if we could make this dependency optional, i.e. to try importing it, and if it fails to just use the default matplotlib ``hist``

e.g.
```python
try:
    from astropy.visualization import hist
except ImportError:
    from matplotlib.pyplot import hist
```

Reply:
That would work. Should specify in the documentation which keywords go with the astropy version then. Won't be able to use 'knuth' as default for 'bins' anymore, though.

Reply:
In general, it's a good idea to not override the defaults unless there's a good reason not to.

It looks like the default for astropy is the same as matplotlib and numpy (``bins=10``). Your extension allows the option of ``knuth`` to be passed through (which you prefer), but you should remove the overriding defaults that you have here.

Reply:
modified accordingly